gossip3to4/addblockrequest.go

```diff
@@ -73,7 +75,7 @@ func ConvertABR(abr *g3services.AddBlockRequest) (*g4services.AddBlockRequest, e
 		ObjectId:    abr.ObjectId,
 		PreviousTip: abr.PreviousTip,
 		Height:      abr.Height,
-		NewTip:      abr.NewTip,
+		NewTip:      []byte(gossip.MagicValidNewTip), // just for dark traffic testing
```

Comment:
this magic new tip will fail the next step https://github.com/quorumcontrol/tupelo/blob/master/gossip/node.go#L508 though maybe we just don't care for now.

Reply:
Will it fail or will it be magic tips all the way down?